01week/resume/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE>
```

Comment:
You have excellently structured HTML in this document. It is clear, logical, and I am able to understand this page without have to look at CSS. Great job on your HTML!